pkg/mcproducer/templates/pull-image.gotmpl

```diff
@@ -1,13 +1,22 @@
 #!/bin/bash
```

Comment:
I believe it is systemd calling those scripts, right? If yes, I think it would be cleaner if the variables were either environment variables or arguments configured in the unit file.

Reply:
Not sure i agree: those are one-shot services. We don not expect to configure/change them once they have been run. In addition having all the parameters defined in the scripts allows for easier testing of scripts as standalone